### ISSUE TITLE
VCR config.ignore_localhost = true

### ISSUE THREAD

**Issue by eliotsykes** [oesmith/puffing-billy#99]
I've been using puffing-billy in a project that also uses VCR+WebMock. The following config avoided any problems:

```ruby
VCR.configure do |config|
  config.ignore_localhost = true
end
```

This is different to the config suggested in the README. Would a pull-request with this be welcome? If yes should this config be in addition or instead of the existing VCR+WebMock config given in the README?

**Comment by ronwsmith** [comment]
If your test needs both, it's typically doing too much, though I have heard success stories using both when necessary.  If you want to write up a blog post or something with your full config, I'd be happy to link to it from `Resources`.

**Comment by eliotsykes** [comment]
Sorry @ronwsmith I didn't have enough of an appreciation for what it was needed for. I've just encountered a situation where the `config.ignore_localhost = true` setup wasn't enough. I needed something similar to the config suggested in the README.

```ruby
RSpec.configure do |config|
  config.around(:each, http_proxy: :puffing_billy) do |example|
    WebMock.allow_net_connect!
    VCR.turned_off { example.run }
    WebMock.disable_net_connect!
  end
end
```

The only change I used was using metadata that uses an option like `http_proxy: :puffing_billy`. Otherwise identical. Documenting here in case its of help to others.
